cxx/distributions/zero_mean_normal.cc

```diff
@@ -37,20 +37,20 @@ void ZeroMeanNormal::unincorporate(const double& x) {
 double ZeroMeanNormal::logp(const double& x) const {
   // Equation (119) of https://www.cs.ubc.ca/~murphyk/Papers/bayesGauss.pdf
   double alpha_n = alpha + N / 2.0;
-  double beta_n = beta + var * N;
+  double beta_n = beta + 0.5 * var * N;
   double t_variance = beta_n / alpha_n;
   return log_t_distribution(x, 2.0 * alpha_n, t_variance);
 }
 
 double ZeroMeanNormal::logp_score() const {
   // Marginal likelihood from Page 10 of
-  // https://j-zin.github.io/files/Conjugate_Bayesian_analysis_of_common_distributions.pdf
+  // https://people.eecs.berkeley.edu/~jordan/courses/260-spring10/lectures/lecture5.pdf
```

Comment:
Is there a reason you like this reference better?  I prefer the old one because it has the explicit formula for the marginal likelihood of the data.

Reply:
It's incorrect because it has a factor of 2 on the beta (caught by the marginalization tests).